portal/src/views/projects/ProjectPublic.vue

```diff
@@ -82,7 +80,65 @@
             <ProjectStations :project="project" :admin="false" :userStations="userStations" />
         </div>
 
-        <ProjectActivity :displayProject="displayProject" v-if="false" />
+        <div class="project-team-activity">
```

Comment:
We should be styling the ProjectActivity component instead of having styled content copy and pasted from the designs.

Reply:
I usually componentize once the data flow is decided.
Couldn't find the needed data in the parent component or a separate api.